### PR TITLE
Guide suggestions

### PR DESCRIPTION
**Pull request by microcosm**
These are mostly stylistic & content updates - readability, explanation, layout, and also a URL change

### CODE REVIEW

File: guide/index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        </div>
+    </header>
+
+    <section id="about-guide" class="sub-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-10 col-lg-offset-1">
+                    <div class="text">
+                        <h2>What this is</h2>
+                        <p>This project is an ongoing effort to provide guidance to tech professionals on navigating the climate crisis. It is designed to be useful to people in and outsode of tech firms who work in tech, and to help you leverage your position and navigate to make positive change within your company, organization or in general society.</p>
```
Comment (mrchrisadams) [inline guide/index.html]: I think this is a typo here - _outsode_ vs _outside_

File: guide/index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header page-scroll">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
+                </button>
+                <a class="navbar-brand" href="#page-top">ClimateAction.tech</a>
+            </div>
+
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```
Comment (mrchrisadams) [inline guide/index.html:69]: Given that this this is looking in okay shape now, how about we add it to the menu? There's no way to access this guide page without knowing the `/guide` url in advance.

**Review by mrchrisadams — CHANGES_REQUESTED** [review]
Hi Andy/@microcosm, this looks good - thanks for the updates! 

Just saw a tiny typo, and had a question about adding this to the main nav, as I figured making this more visible might be an incentive to increase the no. of submissions, and make it easier to find the content for people new to CAT.

File: guide/index.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="page-scroll">
+                        <a href="#about">About</a>
```
Comment (melissahsiung) [inline guide/index.html]:
```suggestion
                        <a href="/#about">About</a>
```

File: guide/index.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    <title>ClimateAction.tech</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link href="/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Theme CSS -->
+    <link href="/css/freelancer-3.css" rel="stylesheet">
+
+    <!-- Custom Fonts -->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
```
Comment (melissahsiung) [inline guide/index.html]:
```suggestion
    <link href="/vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
```

File: guide/index.html (hunk 3 above)
Comment (melissahsiung) [reply guide/index.html]: (and the other menu items as well)

File: guide/index.html [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+    <section id="about-guide" class="sub-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-10 col-lg-offset-1">
+                    <div class="text">
+                        <h2>What this is</h2>
+                        <p>This project is an ongoing effort to provide guidance to tech professionals on navigating the climate crisis. It is designed to be useful to people in and outsode of tech firms who work in tech, and to help you leverage your position and navigate to make positive change within your company, organization or in general society.</p>
+                        <p>The project is currently at it's first (Alpha 1.0) stage, which includes ann organized list of helpful resources which can be easily filtered by category, and contributed to.</p>
+                        <p>This is just the beginning. In future versions, we plan to have curated journeys through resources for people in different positions and companies at different stages on their climate journies, with explainers and how-tos for specific actions.</p>
```
Comment (melissahsiung) [inline guide/index.html]:
```suggestion
                        <p>This is just the beginning. In future versions, we will include action plans, with explainers, how-tos, and resources, for people in different positions in companies at different stages of climate sustainability.</p>
```

File: guide/index.html (hunk 1 above)
Comment (microcosm) [reply guide/index.html]: Fixed in 5e55c4a47862bb8e7af5195b7ba639a4dffaf7d3

File: guide/index.html (hunk 3 above)
Comment (microcosm) [reply guide/index.html]: Ah good spot @melissahsiung. Also I notice you have done something similar in b9cbbcef0cfdc40ace48a4a9472ab6baf8abedb3

But we shouldn't put the `/` on `#page-top` or `#maincontent` as these IDs exist on both pages. They just help users navigate to the to top of the page / content.

I'll fix this now in my PR

Comment (microcosm) [reply guide/index.html]: OK I think I've got these covered here 223f079

File: guide/index.html (hunk 4 above)
Comment (microcosm) [reply guide/index.html]: And here 223f079

File: guide/index.html (hunk 2 above)
Comment (microcosm) [reply guide/index.html:69]: Done in ef84ca0

**Comment by microcosm** [comment]
Ready for review again @mrchrisadams @melissahsiung

**Review by mrchrisadams — APPROVED** [review]

**Pull request merged**